### PR TITLE
Readme modified

### LINKED ISSUE #211: Readme file enhancement
The Readme file can be enhacned by adding a list of all the OPen Source programs in which this project was listed in.

### PR DESCRIPTION
**Pull request by anirudhsai20**
# Description
I have added a list of all the Open Source Programs in which this project was listed in.

Fixed #211 

# Checklist:

- [x] I have pulled the latest changes in the repository
- [ ] I have squashed my commits
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,6 +114,16 @@ To get started have a look at our [documentation on contributing](https://github
 
 Code released under the [MIT License](https://github.com/Girl-Code-It/Opportunity-Calendar-Frontend/blob/develop/LICENSE).
 
+## Opportunity-Calendar-Frontend is a part of these Open Source programs
```
Comment (vaishali614) [inline README.md]: 1. Shift this section above Table of contents
2. Change the heading to 'We're a part of these Open Source Programs 
3. Center align the heading

**Review by vaishali614 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (vaishali614) [reply README.md]: @anirudhsai20 Any updates?

Comment (anirudhsai20) [reply README.md]: I will update my PR within today.

**Comment by anirudhsai20** [comment]
I am very sorry for the delay. I was ties up with my college work. I have made the necessary changes.

**Comment by vaishali614** [comment]
> I am very sorry for the delay. I was ties up with my college work. I have made the necessary changes.

Changes look good now. Please take the latest pull to get it merged.

**Review by Manvityagi — APPROVED** [review]

**Pull request merged**